doc/user/sql/create-source/avro-file.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "CREATE SOURCE: Avro from local file"
+description: "Learn how to connect Materialize to an Avro-formatted local file"
+menu:
+  main:
+    parent: 'create-source'
+---
+
+`CREATE SOURCE` connects Materialize to some data source, and lets you interact
+with its data as if it were in a SQL table.
+
+This document details how to connect Materialize to an Avro-formatted local
+file. For other options, view [`CREATE  SOURCE`](../).
```

Comment:
We, I believe, intend to support other formats for avro files (e.g. json-encoded and line-delimited) I would consider specifically calling this file `avro-ocf` and saying something like this here:

```markdown
This document details how to connect Materialize to an [Avro Object Container 
File](https://avro.apache.org/docs/current/spec.html#Object+Container+Files). 
For other options, view [`CREATE SOURCE`](../).
```